stylesheets/styles.css

```diff
@@ -4,7 +4,6 @@
 
 .wrapper {
   display: -webkit-box;
-  display: -moz-box;
```

Comment:
guess we don't need this anymore... unless we still want to support super outdated FF?